next.config.js

```diff
@@ -26,6 +29,49 @@ module.exports = withCSS(withSourceMaps({
     APP_VERSION: packageJson.version,
     UNLY_SIMPLE_LOGGER_ENV: process.env.APP_STAGE, // Used by @unly/utils-simple-logger - Fix missing staging logs because it believes we're in production
   },
+  experimental: {
+    redirects() {
+      const redirects = [
+        {
+          // Redirect root link with trailing slash to non-trailing slash, avoids 404 - See https://github.com/zeit/next.js/discussions/10651#discussioncomment-8270
+          source: '/:locale/',
+          destination: '/:locale',
+          permanent: process.env.APP_STAGE !== 'development', // Do not use permanent redirect locally to avoid browser caching when working on it
+        },
+        {
+          // Redirect link with trailing slash to non-trailing slash (any depth), avoids 404 - See https://github.com/zeit/next.js/discussions/10651#discussioncomment-8270
+          source: '/:locale/:path*/',
+          destination: '/:locale/:path*',
+          permanent: process.env.APP_STAGE !== 'development', // Do not use permanent redirect locally to avoid browser caching when working on it
+        },
+      ];
+
+      if (process.env.APP_STAGE === 'development') {
+        console.info('Using experimental redirects:', redirects);
+      }
+
+      return redirects;
+    },
+    rewrites() {
+      const rewrites = [
+        {
+          // XXX Doesn't work locally (maybe because of rewrites), but works online
+          source: '/',
+          destination: '/api/autoRedirectToLocalisedPage',
```

Comment:
Isn't `/autoRewriteToLocalisePage` a better name since this is a `rewrite` and not `redirect` after all?

Reply:
Good catch lmao, I don't know. To me it makes sense as "redirect" because it redirects to the localised page, the rewrite is the first part, what matters is the redirect that happens on the node.js API IMHO. I believe it makes more sense to leave it as it.

Reply:
I'm ok with that.